client/components/quizList/quizList.jsx

```diff
@@ -17,10 +16,21 @@ const QuizList = ({ name, category, difficulty }) => {
       .catch(err => console.error(err))
   };
 
+  const axiosGetQuizzesByRandomSelection = () => {
```

Comment:
this function is named the same as the function above. Did you mean to name it "axiosGetQuizzesBySelection" instead?